test/controllers/movies_controller_test.rb

```diff
@@ -0,0 +1,93 @@
+require "test_helper"
+
+describe MoviesController do
+  def check_response(expected_type:, expected_status: :success)
+    must_respond_with expected_status
+    expect(response.header['Content-Type']).must_include 'json'
+
+    body = JSON.parse(response.body)
+    expect(body).must_be_kind_of expected_type
+    return body
+  end
```

Comment:
Since this helper method is in multiple files, you could probably bring it out into the `test_helper` file